src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelSummary/SummaryRow.tsx

```diff
@@ -1,65 +1,16 @@
 import React from "react";
-import classNames from "classnames";
 
-import MaterialIcon from "ui/components/shared/MaterialIcon";
-import escapeHtml from "escape-html";
-const { getCodeMirror } = require("devtools/client/debugger/src/utils/editor/create-editor");
-
-interface SummaryRowProps {
-  isEditable: boolean;
-  handleClick: (event: React.MouseEvent) => void;
+export default function SummaryRow({
+  expression,
```

Comment:
If there's only one prop that accepts a `ReactNode`, we should consider making it be `children` since it lends itself to more readable code.

`<MyThing coolProp={<span>Hi!</span>}>` vs `<MyThing><span>Hi!</span></MyThing>`